ROX:26971: Add CRS to ACS docs (#89252)
openshift/openshift-docs: modules/installing-rhacs-kubernetes-steps.adoc

. Add the {product-title-short} Helm charts repository.
. Install the `central-services` Helm chart on the cluster that will contain Central, called the Central cluster.

> . Log in to the {product-title-short} web console from the Central cluster and create an init bundle or cluster registration secret (CRS) that you will install on the cluster that you want to secure, called the secured cluster. The init bundle or the CRS contains the artifacts that {product-title-short} uses to establish trusted connections between Central and secured clusters.

Reviewer: Maybe something like...

```suggestion
. Log in to the {product-title-short} web console from the Central cluster and create an init bundle or cluster registration secret (CRS) that you need to apply to the cluster that you want to secure, called the secured cluster. The init bundle or the CRS contains the artifacts that {product-title-short} uses to establish trusted connections between Central and secured clusters.
```

?

Personally I would rather use `install` for applications/code that is executing in some form or another, and `apply` for manifests that purely contain data. But maybe that is just me.

> . Log in to the {product-title-short} web console from the Central cluster and create an init bundle or cluster registration secret (CRS) that you will install on the cluster that you want to secure, called the secured cluster. The init bundle or the CRS contains the artifacts that {product-title-short} uses to establish trusted connections between Central and secured clusters.

Reviewer: Maybe throw in an `cryptographic` before the `artifact`?

+
--
:FeatureName: Cluster registration secret
include::snippets/technology-preview.adoc[]
--
. For each secured cluster, install the `secured-cluster-services` Helm chart by running one of the following commands:
* For an init bundle, run the following command:
+
[source,terminal]
----
$ helm install -n stackrox \
  --create-namespace stackrox-secured-cluster-services rhacs/secured-cluster-services \

>   -f <name_of_cluster_init_bundle.yaml> \

Reviewer: Is it on purpose that we don't have a number attached to the end of this line with an explanation as we are having for the other settings?

Author: No, just that the text for that would be "name of the cluster init bundle" and typically the number with the explanation is for things that need additional clarification.

>   -f <path_to_values_public.yaml> -f <path_to_values_private.yaml> \// <1>

Reviewer: I guess I would find it easier to read if there was a line break between these two settings.

  --set imagePullSecrets.username=<username> \// <2>
  --set imagePullSecrets.password=<password> <3>
----
<1> Use the `-f` option to specify the paths for your YAML configuration files.
<2> Include the user name for your pull secret for Red{nbsp}Hat Container Registry authentication.
<3> Include the password for your pull secret for Red{nbsp}Hat Container Registry authentication.
* For a CRS, run the following command:
+
[source,terminal]
----
$ helm install -n stackrox \
  --create-namespace stackrox-secured-cluster-services rhacs/secured-cluster-services \

>   --set-file crs.file=<crs_file_name> \

Reviewer: Maybe add a `.yaml` suffix here as well for consistency?

>   --set-file crs.file=<crs_file_name> \

Reviewer: Same here: Is it on purpose that we don't have a number attached to the end of this line with an explanation as we are having for the other settings?

Author: I should've put one here as I did in a previous section--added.

>   -f <path_to_values_public.yaml> -f <path_to_values_private.yaml> \// <1>

Reviewer: I guess I would find it easier to read if there was a line break between these two settings.
